sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -323,12 +330,17 @@ export class KeysClient {
       delete unflattenedOptions.expires;
       delete unflattenedOptions.requestOptions;
 
+      const span = this.createSpan("createEcKeyMethod", unflattenedOptions);
+      span.start();
+
       const response = await this.client.createKey(
         this.vaultBaseUrl,
         name,
         options.hsm ? "EC-HSM" : "EC",
         unflattenedOptions
       );
+
+      span.end();
```

Comment:
We also need to end the span if `createKey` throws an error, otherwise the span will never be closed.

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/4654/commits/53a6c66fc5563484a7bafbc60082df81cae0362b